src/Event.php

```diff
@@ -178,6 +187,30 @@ public function setSdkIdentifier(string $sdkIdentifier): void
         $this->sdkIdentifier = $sdkIdentifier;
     }
 
+    /**
+     * Gets the version of the SDK package that generated this Event.
+     *
+     * @return string
+     */
+    public function getSdkVersion(): string
+    {
+        if (null === $this->sdkVersion) {
+            $this->sdkVersion = PrettyVersions::getVersion('sentry/sentry')->getPrettyVersion();
+        }
+
+        return $this->sdkVersion;
+    }
+
+    /**
+     * Sets the version of the SDK package that generated this Event.
+     *
+     * @param string $sdkVersion
```

Comment:
```suggestion
     * @param string $sdkVersion
     *
     * @internal
```

Let's make these functions internal.